ExosIIMountControl.hpp

```diff
@@ -415,11 +444,33 @@ class ExosIIMountControl :
         {
             std::vector<uint8_t> messageBuffer;
             if(SerialDeviceControl::SerialCommand::GetSyncCommandMessage(messageBuffer, rightAscension, declination))
-            {
-                std::cerr << "Sent Sync command!" << std::endl;
-                return SerialDeviceControl::SerialCommandTransceiver<InterfaceType, TelescopeMountControl::ExosIIMountControl<InterfaceType>>::SendMessageBuffer(
-                           &messageBuffer[0], 0, messageBuffer.size());
-                //return rc && mMountStateMachine.DoTransition(TelescopeSignals::GoTo);
+            {                
+                if (true) {  
```

Comment:
If you are sure this code is fine, just remove the old, I don't see a reason for this `if(true)` here.